src/main/java/pacman/ActionKeyPacman.java

```diff
@@ -7,8 +7,14 @@
 public class ActionKeyPacman implements KeyListener {
 
     private DrawComponents drawComponents;
-    private int key;
     public static final int TIMER_PACMAN = 200;
+    public static final String MOVE_LEFT = "left";
+    public static final String MOVE_RIGHT = "right";
+    public static final String MOVE_UP = "up";
+    public static final String MOVE_DOWN = "down";
```

Comment:
Se utilizan estas constantes en alguna parte del codigo? revise pero no vi que las uses al menos en esta clase

Reply:
thank you for your comment, hey are used in the last commit.